.env.template

```diff
@@ -8,6 +8,7 @@ RUN_MIGRATIONS=False
 DEBUG_SIMULATED_LAG=0
 SECRET_KEY=xxx
 ALLOWED_HOSTS=*
+ALLOWED_CDNS="gcdn.thunderstore.io,hcdn-1.hcdn.thunderstore.io"
```

Comment:
Since the `.env.template` is intended mostly for dev use, these can probably be omitted entirely? If the intention was to document prod env settings the readme would be a better place, but I'd also not make it a _requirement_ for this PR (given that this is an optional feature & many other optional options lack docs there; it's mostly for internal use after all)

Reply:
Removed.